### PR TITLE
[#309] Merge annox

### PR DESCRIPTION
**Pull request by mattrpav**
NOTE: po project still not functional due to need for modernization in testing

### CODE REVIEW

**Review by laurentschoelens — COMMENTED** [review]
See comments on some files

File: jaxb-annox-parent/jaxb-annox/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Manifest-Version: 1.0
```
[the remark this answers is not in the document]
Comment (mattrpav) [reply jaxb-annox-parent/jaxb-annox/META-INF/MANIFEST.MF:1]: This is used for making the jar into an OSGi bundle. Another ticket pass to refactor and remove this packaging file and step

File: jaxb-annox-parent/jaxb-annox/.gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+target
```
[the remark this answers is not in the document]
Comment (mattrpav) [reply jaxb-annox-parent/jaxb-annox/.gitignore:1]: No, all the sub-dir .gitignore should be removed/consolidated into the parent .gitignore. A separate ticket for repo clean-up.

File: jaxb-annox-parent/jaxb-annox/pom.xml [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+                        <scope>provided</scope>
+                </dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>commons-io</groupId>
+			<artifactId>commons-io</artifactId>
+ 			<version>2.4</version>
```
[the remark this answers is not in the document]
Comment (mattrpav) [reply jaxb-annox-parent/jaxb-annox/pom.xml:31]: Yes, just trying to get it to build/work. Refactor and upgrade can be follow-on task

**Comment by laurentschoelens** [comment]
LGTM
Go for merge

**Pull request merged**